lab-jonathan/lib/readHex.js

```diff
@@ -0,0 +1,51 @@
+let fs = require('fs');
+let oneString;
+let twoString;
+let threeString;
+let allData = [];
+
+// function otherFunction(data) {
+//   console.log(data)
+// }
+//
+// var file = fs.readFile('blah')
+// otherFunction(file)
+//
+// fs.readFile('blah.txt', function(data) {
+//   return data
+//   otherFunction(data)
+// })
+
+
+//so i'm going to, instead of running a log at the end of each function, i'm going to push that specific file's data into an array.  then at the very end i'm going to call my callback, which is actually the testing function.
+
+function readHex(cb) {
+  fs.readFile('./lib/one.txt', function(err, data){
+    if(err) throw err;
+    oneString = data.toString();
```

Comment:
Go take a look at the Node (not MDN) docs for `toString()`. You can eliminate the need for both `oneString` and `dataOne` by directly pushing `data.toString()` to your array. This follows through to the other nested callbacks as well.